src/v2/Components/__stories__/Editorial.story.tsx

```diff
@@ -74,15 +73,14 @@ export const TextCentered = () => {
   )
 }
 
+// TODO: Do we need a proper sticky library?
```

Comment:
@damassi I noticed this when updating. Obviously this wasn't working previously. I started to implement what we needed to fix this but at that point realized I was writing a fully-fledged sticky library. I don't necessarily have a problem with doing this; particularly if we can't find a decent one that uses intersection observer and has typings... but figured we should just cross this bridge when we actually need it.

Reply:
No prob at all, all this lib code you've been writing as of late is A+